# Unique session id
int32 session_id

# An action provided by the environment (in json format)
string action

---

# Task specific observation (in json format)
string observation

# Amount of reward returned after previous action
float32 reward

# Whether the episode has ended
bool done

# Contains auxiliary diagnostic information (in json format)
string info

---
